docs/source/noise/column_noise.rst

```diff
@@ -19,6 +19,30 @@ Misreport age
 Write the wrong digits
 ----------------------
 
+Sometimes people may write the wrong number for numeric data such as a street
+number, date, or social security number. This could be intentional or
+accidental. pseudopeople simulates this type of noise in numeric columns by
+randomly selecting some digits from the true number and replacing each selected
+digit with a diferent digit selected uniformly at random.
+
+This noise type is called :code:`write_wrong_digits` in the configuration.
+It takes two parameters:
+
+.. list-table:: Parameters to the write_wrong_digits noise type
+  :widths: 1 5 1
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default
+  * - :code:`cell_probability`
+    - The probability that any given cell in the column will be selected to be eligible for this type of noise.
+    - 0.01 (1%)
+  * - :code:`token_probability`
+    - The conditional probability, given that a numeric cell has been selected for noise eligibility, that any given digit in the true number will be replaced by a different digit.
```

Comment:
This is a bit inconsistent with how I've been writing this, but I think it is clear. Maybe we should do a consistency pass at some point when all the noise types are documented, but I think it's fine for now.

Reply:
Yeah, this was one of the main things I wanted your opinion on, to see if we can settle on a preferred way to describe these things. Maybe I can try editing to match yours more closely...

Reply:
Just leaving it for now in the interest of getting it merged. We can take another pass later.